FrontEnd/src/Components/ImageCarousel.jsx

```diff
@@ -0,0 +1,157 @@
+import { Box, Image, HStack, Text, Flex } from "@chakra-ui/react";
+import { useState } from "react";
+
+// const ImageCarousel = () => {
+//   return (
+//     <Box>
+//       <Flex w="full" overflow="hidden" pos="relative">
+//         <Flex h="400px" w="full" {...carouselStyle}>
+//           {slides.map((slide, sid) => (
+//             <Box key={`slide-${sid}`} boxSize="full" shadow="md" flex="none">
+//               <Image
+//                 src={slide.img}
+//                 alt="carousel image"
+//                 boxSize="full"
+//                 backgroundSize="cover"
+//               />
+//             </Box>
+//           ))}
+//         </Flex>
+//         <Text {...arrowStyles} left="0" onClick={prevSlide}>
+//           &#10094;
+//         </Text>
+//         <Text {...arrowStyles} right="0" onClick={nextSlide}>
+//           &#10095;
+//         </Text>
+//       </Flex>
+//       <HStack
+//         justify="center"
+//         pos="absolute"
+//         bottom="12px"
+//         w="full"
+//         spacing="2"
+//       >
+//         {Array.from({ length: slidesCount }).map((_, slide) => (
+//           <Box
+//             key={`dots-${slide}`}
+//             cursor="pointer"
+//             boxSize={["4px", null, "7.5px"]}
+//             bg={currentSlide === slide ? "blackAlpha.800" : "blackAlpha.500"}
+//             rounded="50%"
+//             display="inline-block"
+//             transition="background-color 0.6s ease"
+//             _hover={{
+//               bg: "blackAlpha.800",
+//             }}
+//             onClick={() => setSlide(slide)}
+//           ></Box>
+//         ))}
+//       </HStack>
+//     </Box>
+//   );
+// };
+
+// export default ImageCarousel;
+
+const ImageCarousel = () => {
+  const arrowStyles = {
+    cursor: "pointer",
+    pos: "absolute",
+    top: "50%",
+    w: "auto",
+    mt: "-22px",
+    p: "16px",
+    color: "white",
+    fontWeight: "bold",
+    fontSize: "18px",
+    transition: "0.6s ease",
+    borderRadius: "0 3px 3px 0",
+    userSelect: "none",
+    _hover: {
+      opacity: 0.8,
+      bg: "black",
+    },
+  };
+  const slides = [
+    {
+      img: "https://images.pexels.com/photos/2599537/pexels-photo-2599537.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940",
+    },
+    {
+      img: "https://images.pexels.com/photos/2714581/pexels-photo-2714581.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940",
+    },
+    {
+      img: "https://images.pexels.com/photos/2878019/pexels-photo-2878019.jpeg?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260",
+    },
+    {
+      img: "https://images.pexels.com/photos/1142950/pexels-photo-1142950.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940",
+    },
+    {
+      img: "https://images.pexels.com/photos/3124111/pexels-photo-3124111.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940",
```

Comment:
create a js file and import the mock data

Reply:
Done 👍